compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -634,6 +634,9 @@ error.undefined.parameter=\
 error.invalid.error.reason.type=\
   invalid error reason type ''{0}'', expected a subtype of ''string''
 
+error.error.match.over.const.reason.ref.not.supported=\
+  error match pattern with reason as a ''constant'' reference is currently not supported
```

Comment:
How about something like 
```
matching error reason to a ''const'' pattern is currently not supported.
```
?

Reply:
Matching to a const literal is supported